bugsnag/utils.py

```diff
@@ -123,7 +128,7 @@ def _sanitize_dict_key_value(self, clean_dict, key, clean_value):
                     'Could not add sanitize key for dictionary, '
                     'dropping value.')
 
-    def _sanitize_dict(self, obj, trim_strings, ignored):
+    def _sanitize_dict(self, obj, trim_strings, ignored, seen):
```

Comment:
yeah its intentional, since filtering does its own "ignored" checking as its available at the top level. Though it should also do its own "seen" checking!